Source/Redemption/Private/AudioManager.cpp

```diff
@@ -65,8 +65,9 @@ void AAudioManager::StartRandomAmbientSounds()
 // Implementation to start playing ambient music
 void AAudioManager::PlayAmbientMusic()
 {
-    if (AmbientMusicCue != nullptr && !bIsInCombat)
+    if (AmbientMusicCue != nullptr && bIsInCombat)
     {
+        bIsInCombat = false;
```

Comment:
Can you explain me why did you change this?

Reply:
the bug before was that when we create the timer to go back to safe music we change the In combat to be false, and if we entered the vision of a rat again it would be false and because of that it would restart the music. To solve the bug I changed where it changes the bool variable

Reply:
Understood. Thank you for the explanation.